clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -296,6 +301,11 @@
                                         9 * 60 * 1000,
                                         Importance.MEDIUM,
                                         CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_DOC)
+                                .define(INTERCEPTOR_CLASSES_CONFIG,
+                                        Type.LIST,
+                                        "",
```

Comment:
Now I see why everyone is passing "" for the empty list. Will need to fix AbstractConfig.getConfiguredInstances as well. Will do.